python.sh

```diff
@@ -3,52 +3,63 @@ version: "%(tag_basename)s"
 tag: v3.9.12
 source: https://github.com/python/cpython
 requires:
- - AliEn-Runtime:(?!.*ppc64)
- - FreeType
- - libpng
- - sqlite
- - "GCC-Toolchain:(?!osx)"
- - libffi
+  - zlib
+  - "OpenSSL:(?!osx)"
+  - "osx-system-openssl:(osx.*)"
+  - AliEn-CAs
+  - FreeType
+  - libpng
+  - sqlite
+  - libffi
 build_requires:
- - curl
- - alibuild-recipe-tools
+  - alibuild-recipe-tools
+  - "GCC-Toolchain:(?!osx)"
+  - "Xcode:(osx.*)"
 env:
-  SSL_CERT_FILE: "$(export PATH=$PYTHON_ROOT/bin:$PATH; export LD_LIBRARY_PATH=$PYTHON_ROOT/lib:$LD_LIBRARY_PATH; python -c \"import certifi; print(certifi.where())\")"
+  SSL_CERT_FILE: "$(PATH=$PYTHON_ROOT/bin:$PATH \
+                    LD_LIBRARY_PATH=$PYTHON_ROOT/lib:$LD_LIBRARY_PATH \
+                    python3 -c 'import certifi; print(certifi.where())')"
```

Comment:
@adriansev for the builders that fail after Python, the error is:

```
2023-03-17@08:05:38:DEBUG:O2Suite:Python:build_O2_alidist-slc7_x86-64: /build.sh: line 218: syntax error near unexpected token `certifi.where'
```

You need to go back to using `\"` for the inner quotes I think... aliBuild's escaping is sometimes nonexistent, so this variable assignment gets turned into the following line in `build.sh`:

```
echo 'export SSL_CERT_FILE="$(PATH=$PYTHON_ROOT/bin:$PATH LD_LIBRARY_PATH=$PYTHON_ROOT/lib:$LD_LIBRARY_PATH python3 -c 'import certifi; print(certifi.where())')"' >> $INSTALLROOT/etc/profile.d/init.sh
```

But really, this is a bug in aliBuild. I'll see if I can fix the escaping in future, but for now, it might be better to go back to `\"`, sorry!

Reply:
done! let's see how it goes :)